XenoCP

adamdingliang opened the pull request:
Documentation for XenoCP pipeline.

claymcleod commented:
Also, Adam, we need to ensure that the .idea files aren’t added to the repo. Please remove them before merging.

> On Apr 29, 2019, at 5:06 PM, mtreadwa <notifications@github.com> wrote:
> 
> @mtreadwa requested changes on this pull request.
> 
> Ling in the inputs table did you mean "buckets"?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

adamdingliang commented:
Fixed the typos and removed .idea files.

adamdingliang commented:
@agout thanks for the comments. Responses to the questions:
1. yes, paired-reads.
2. "Mate suffix length" is read name suffix length. The split step of the workflow also trims the unnecessary read name suffixes such as /1 and /2. It allows users to choose the suffix length to trim. I have modified the description to "Length of read name suffixes to be trimmed. [default: 3]". Hope it is clear.
3. Updated the description to "Number of buckets to split original bam to for parallelism [default: 31]"
4. Updated the workflow image.
5. There are rules to determine it aligns better to human or mouse. I don't think we can put rules in the diagram. So no good idea of how to make it better. Any further suggestion?
6. Yes. Change to LCRs to be clear.

agout commented:
thanks Adam and Madison

adamdingliang commented:
Thanks very much, Alex and Madison.
